src/libraries/System.Globalization/tests/System/Globalization/RegionInfoTests.cs

```diff
@@ -177,13 +177,18 @@ public void ISOCurrencySymbol(string name, string expected)
             Assert.Equal(expected, new RegionInfo(name).ISOCurrencySymbol);
         }
 
-        [Theory]
-        [InlineData("en-US", new string[] { "$" })]
-        [InlineData("zh-CN", new string[] { "\u00A5", "\uffe5" })] // \u00A5 is Latin-1 Supplement(Windows), \uffe5 is Halfwidth and Fullwidth Forms(ICU)
-        public void CurrencySymbol(string name, string[] expected)
+        [Fact]
+        public void CurrencySymbol()
         {
-            string result = new RegionInfo(name).CurrencySymbol;
-            Assert.Contains(result, expected);
+            Assert.Equal("$", new RegionInfo("en-US").CurrencySymbol);
+            if (PlatformDetection.IsNotBrowser)
+            {
+                Assert.Contains(new RegionInfo("zh-CN").CurrencySymbol, new string[] { "\u00A5", "\uffe5" });
```

Comment:
Usually the expected value goes at the left.

Reply:
I didn't change the order here, it's Assert.Contains -- we check CurrencySymbol to contain a specific sequence.